packages/main/src/plugin/util/exec.spec.ts

```diff
@@ -430,6 +430,48 @@ describe('exec', () => {
     expect(spawnMock).not.toHaveBeenCalled();
     expect(sudo.exec).toBeCalledWith('echo Hello, World!', expect.anything(), expect.anything());
   });
+
+  test('should run the command with privileges on Windows and remove unsupported environment', async () => {
+    const command = 'echo';
+    const args = ['Hello, World!'];
+    (util.isWindows as Mock).mockReturnValue(true);
+    let options: any;
+
+    (sudo.exec as Mock).mockImplementation((_command, _options, callback) => {
+      callback(undefined);
+      options = _options;
+    });
+
+    vi.mock('child_process', () => {
```

Comment:
This vi.mock call overrides all previous calls. It is luck they are all the same so nothing is affected.
It can be moved on top of the file with other mocks and removed from all the tests in this file.